### DISCUSSION TITLE
Example of how to use mimir with filesystem as storage

### DISCUSSION THREAD

**Discussion by alita1991** [grafana/mimir#6735] [General]
Hi,

I'm looking for an example of how to use mimir with filesystem example, I'm not unable to find this documented anywhere.

Any help would be highly appreciated.
Thanks

**Comment by masterwishx** [comment]
check : 
https://github.com/grafana/mimir/blob/main/docs/configurations/demo.yaml
https://github.com/grafana/mimir/blob/main/docs/configurations/single-process-config-blocks.yaml
